yuhow/Makefile

```diff
@@ -1,76 +0,0 @@
-SHELL := bash
```

Comment:
Why do you remove the make file?  Usually an automatic build system would help people understand how to build.  Also, this make file takes care of toc building and keeps build logs.  Would you please describe why you don't want to use the automatic build script?